packages/indexer/src/core/indexer.ts

```diff
@@ -126,7 +126,9 @@ export class Indexer {
     this.domain = domain;
     this.sdk = sdk;
     this.orchestrator = orchestrator;
-    this.develop = false;
+    this.develop = !!(
```

Comment:
Might be able to do `this.develop = process.env.NODE_ENV === 'development'` here. Also do we need staging specific logic as well or just for development?

Reply:
yes, thanks! :)